Libraries/Geolocation/Geolocation.js

```diff
@@ -68,6 +68,10 @@ type GeoOptions = {
  * Android API >= 18 Positions will also contain a `mocked` boolean to indicate if position
  * was created from a mock provider.
  *
+ * Android API >= 23 Requires an additional step to check for, and request 
+ * the ACCESS_FILE_LOCATION permission using the 
```

Comment:
Er, yeah, that's correct. Sorry, I haven't had any time to look into this at all the past few weeks.